test/auth/oauth.tests.js

```diff
@@ -531,6 +531,21 @@ describe('OAuthAuthenticator', function() {
         })
         .catch(done);
     });
+    it('should include the Auth0 client secret in the request', function(done) {
+      nock.cleanAll();
+      var request = nock(API_URL)
+        .post(path, function(body) {
+          return body.client_secret === CLIENT_SECRET;
+        })
+        .reply(200);
+      this.authenticator
+        .refreshToken(userData)
+        .then(function() {
+          expect(request.isDone()).to.be.true;
+          done();
+        })
+        .catch(done);
```

Comment:
wouldn't calling `done` imply the test is over successfully? And if so, I don't think we want that for the case where something goes wrong like here that an exception is thrown.

Reply:
Honestly, I'm not totally sure. This is only one of a few tests for `refreshToken` and we're just looking for something specific, that the secret is there.